rdt-open-reader/src/covidIndonesia/java/io/ona/rdt/util/CovidRDTJsonFormUtils.java

```diff
@@ -235,17 +235,18 @@ public void populateRDTDetailsConfirmationPage(WidgetArgs widgetArgs, String dev
         Context context = widgetArgs.getContext();
         if (CovidConstants.FormFields.OTHER_KEY.equals(deviceId)) {
             // reset details when other is selected or device id is blank
-            writeRDTDetailsToWidgets(widgetArgs, context.getString(R.string.unknown_rdt_selected),  "", "");
+            writeRDTDetailsToWidgets(widgetArgs, context.getString(R.string.unknown_rdt_selected),  "", "", "", "");
         } else {
             DeviceDefinitionProcessor deviceDefinitionProcessor = DeviceDefinitionProcessor.getInstance(context, refreshDeviceDefinitionBundle);
-            String deviceDetails = getFormattedRDTDetails(widgetArgs.getContext(), deviceDefinitionProcessor.extractManufacturerName(deviceId),
-                    deviceDefinitionProcessor.extractDeviceName(deviceId));
+            String manufacturer = deviceDefinitionProcessor.extractManufacturerName(deviceId);
+            String deviceName = deviceDefinitionProcessor.extractDeviceName(deviceId);
+            String deviceDetails = getFormattedRDTDetails(widgetArgs.getContext(), manufacturer, deviceName);
```

Comment:
We can move this [here](https://github.com/onaio/rdt-standard/pull/920/files#diff-a67c557fd8621e611406f25aff753de7af64317376e85764a3d58a1275c74725R249) and instead remove deviceDetails from the list of parameters.

Reply:
made some change, is that tine